src/python/pants/backend/python/util_rules/pex_from_targets.py

```diff
@@ -298,7 +298,7 @@ async def pex_from_targets(request: PexFromTargetsRequest, python_setup: PythonS
                         requirements=PexRequirements(all_constraints),
                         interpreter_constraints=interpreter_constraints,
                         platforms=request.platforms,
-                        additional_args=["-vvv"],
```

Comment:
I didn't realize that slipped through review. This should instead already be set via `[pex].verbosity`.

Reply:
I'll refrain from changing an unrelated thing unless that's pick worth too and @benjyw clarifies is there was intent behind this choice.

Reply:
Honestly I have no recollection at all of anything related to this. Whatever makes sense to do here is very likely what should be done. If I had intended to do something non-obvious here I would have commented it, so I assume this is just left over from development or something.

Reply:
Got it. So this at least does not break things, `./pants --pex-verbosity=9 run build-support/bin/changelog.py` is fine. It translates to `pex -vvv -vvvvvvvv`, which is legal.

But given that this is leftover from debugging and not intentional, I think it'd be good to clean up.

Reply:
Will do. The test was not sufficient to actually break pre-fix, so it can ride up with that patch here forthwith.